spec/formatting.md

```diff
@@ -685,60 +685,88 @@ and a U+007D RIGHT CURLY BRACKET `}`.
+> An example of this is formatting the value `-1234.56` as the currency `AED`
+> in the `ar-AE` locale. The formatted value appears like this:
+> ```
+> ‎-1,234.56 د.إ.‏
+> ```
+> The code point sequence for this string, as produced by the ICU4J `NumberFormat` function,
+> includes **U+200F U+200E** at the start and **U+200F** at the end of the string.
+> If it did not do this, the same string would appear like this instead:
+> 
+> ![image](https://github.com/unicode-org/message-format-wg/assets/69082/6cc7f16f-8d9b-400b-a333-ae2ddb316edb)
 
-> For example, an implementation could provide a `:number` formatting function
-> which would always produce output matching the message's locale,
-> allowing for its formatted string representation to never need isolation.
+A **_bidirectional isolation strategy_** is functionality in the formatter's
+processing of a _message_ that produces bidirectional output text that is ready for display.
 
-Implementations formatting messages as a concatenated string or a sequence of strings
-MUST provide one or more strategies for bidirectional isolation.
-One such strategy MUST behave as follows:
+The **_Default Bidi Strategy_** is a _bidirectional isolation strategy_ that uses
+isolating Unicode control characters around _placeholder_'s formatted values.
+It is intended for use in plain-text strings, where markup or other mechanisms
+are not available.
+Implementations MUST provide the _Default Bidi Strategy_ as one of the 
+_bidirectional isolation strategies_.
+
+Implementations MAY provide others _bidirectional isolation strategies_.
+
+Implementations MAY supply a _bidirectional isolation strategy_ that performs no processing.
+
+The _Default Bidi Strategy_ is defined as follows:
 
 1. Let `msgdir` be the directionality of the whole message,
    one of « `'LTR'`, `'RTL'`, `'unknown'` ».
    These correspond to the message having left-to-right directionality,
    right-to-left directionality, and to the message's directionality not being known.
+1. Let `localedir` be the default directionality of the current locale,
+   one of « `'LTR'`, `'RTL'`, `'unknown'` ».
+   These correspond to the locale having left-to-right directionality,
+   right-to-left directionality, and to the locale's default directionality not being known.
 1. For each _expression_ `exp` in _pattern_:
    1. Let `fmt` be the formatted string representation of the resolved value of `exp`.
    1. Let `dir` be the directionality of `fmt`,
       one of « `'LTR'`, `'RTL'`, `'unknown'` », with the same meanings as for `msgdir`.
-   1. If `dir` is `'unknown'`:
-      1. In the formatted output,
-         prefix `fmt` with U+2068 FIRST STRONG ISOLATE
-         and postfix it with U+2069 POP DIRECTIONAL ISOLATE.
-   1. Else, if `dir` is `'LTR'` and `msgdir` is not `'LTR'`:
-      1. In the formatted output,
+   1. If `dir` is `'LTR'`:
+      1. If `msgdir` and `localedir` are both `'LTR'`
+         in the formatted output, let `fmt` be itself
```

Comment:
I still contend that the reference here to `localedir` and its definition above should be left out.

```suggestion
      1. If `msgdir` is `'LTR'`, let `fmt` be itself.
```

Reply:
I don't quite agree for the reasons I stated earlier. But I made the change anyway. "Your" way will result in a some LTR values being unwrapped when maybe they should be.